ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -135,9 +135,12 @@ object LedgerApiTestTool {
       })
 
     val testsToRun: Iterable[LedgerTestCase] =
-      (if (config.allTests) cases(Tests.all(config))
-       else if (config.included.isEmpty) cases(Tests.default)
-       else cases(Tests.all(config)).filter(matches(config.included)))
+      ((
+        if (config.allTests) cases(Tests.all(config))
+        else if (config.included.isEmpty) cases(Tests.default)
+        else cases(Tests.all(config)).filter(matches(config.included))
+      ) ++
+        cases(Tests.dev).filter(matches(config.included)))
```

Comment:
This logic has become quite complicated, and I can't imagine it's reasonable to expect a user of the test tool to understand it.

Perhaps a separate `--include-dev` flag to include dev tests?

Alternatively, perhaps an `--include-default` flag that is a no-op, but when you use `--include-default --include=FooIT`, it makes it additive.